script/Ciel/Day 10/10_01_CIEL10_1_1.txt

```diff
@@ -252,7 +252,7 @@ Seeing her smile helps soothe the restlessness I feel in regards to Senpai.
 -- Page 52, Offset 27205.
 -- 小言の多い秋葉だが、真剣に遠野志貴の体を案じてくれている事が伝わってきたからだ。
 // Sorry imoutocons, executive decision here
-Akiha may nag a lot, but I can tell that she cares deeply about my health.
+Akiha may chide a lot, but I can tell that she cares deeply about my health.
```

Comment:
Prefer nag in this instance

Reply:
The more I see, the more I prefer nag in **every** instance, but that was my initial stance, anyway.

Reply:
I think it probably works better in a lot of instances, but also I really do not like throwing away flexibility here. The concept is more important than the word used. Especially for a line line this - 

> 学校から家に連絡が入っていればアウトだが、こればかりはもうどうしようもない。
> If the school contacts the mansion it'll be game over for me, but I can't do much about that now.
> 秋葉になんて小言を言われようと、いや、そもそもこっちだってそれどころじゃないのである。
> I wonder what sort of [小言] Akiha would have in store for me... no, there's no use thinking about something I can't change.

Chide is weak as hell, and nag is just the wrong verb here. He's in for a tongue lashing, not a reminder.

Same for this one -

> 「いえいえ、秋葉さまでしたらお部屋の方に戻られていますので、どうかご安心くださいな。お小言は明日の朝にまで持ち越しというコトで」
> "No need to fear, Akiha-sama has already retired for the evening. Your [小言] has been postponed until tomorrow morning."

Your nag sesh has been postponed? Doesn't really fit imo. Words aren't 1:1, and the JP one here sure seems more flexible than anything we have.

Reply:
'Your round of nagging'
Regardless, yeah, it would be easier to make it all inconsistent, wouldn't it?
But that's besides the point. You should be able to see how incredibly consistent they are with calling Akiha's bitch-outs "小言" at this point. If we don't preserve that, it would be a disfavor.

Maybe a simple 'scolding' works best, but it's a bit generic. I still think nagging can fit in most context if you employ some creativity.

Reply:
I don't agree, but I've done what I could. Take another look over and check.